### ISSUE TITLE
Cannot get this working in Android Studio

### ISSUE THREAD

**Issue by GoogleCodeExporter** [haibison/android-lockpattern#4]
```
What steps will reproduce the problem?
1.import module into android studio


What is the expected output? What do you see instead?
I expect to see modules to import however none are present. My guess is because 
it's missing a build.gradle file

What version of the product are you using? On what operating system?
version 2.8 on Windows 8

Please provide any additional information below.


```

Original issue reported on code.google.com by `joshdo...@gmail.com` on 7 Jul 2013 at 1:09

**Comment by haibison** [comment]
2+ years and Android Studio still doesn't support _stand-alone_ library projects. If you can, please help us by voting for this issue: <https://code.google.com/p/android/issues/detail?id=105570>

**Comment by jagloro** [comment]
> 1. File -> New module -> AndoridLibrary
   Uncheck 'Create activity' and in the 'Package name' enter 'com.haibison.android.lockpattern'. Enter 'Module name' as LockPattern
   -> Finish
2. Copy code/src/com/haibison/android/* to 
LockPattern/src/main/java/com/haibison/android
3. Copy code/res/* to LockPattern/src/main/res
4. Copy code/AndoridManifest.xml to LockPattern/src/main/

This worked for me and probably should be added to the Quick Use Document
Also this needs to be added to the target module's dependencies (build.gradle):
compile project(':LockPattern')

**Comment by haibison** [comment]
@jagloro, I'm sorry that's not a _stand-alone_ project.

**Comment by haibison** [comment]
Support for Android Studio will be available in next stable release (current is `3.2`).

Thanks you guys for waiting so long. I'm sorry.
